### PR TITLE
Added a few tracking Calls

### PR DESCRIPTION
**Pull request by gazzdingo**
### Features and Changes
Lukes request For tracking

- Closes **(add link to issue here)**

### CODE REVIEW

File: packages/front-end/pages/power-calculator/index.tsx [hunk 1]
```diff
@@ -115,6 +115,16 @@ const PowerCalculationPage = (): React.ReactElement => {
         <PowerCalculationSettingsModal
           close={() => setShowModal(false)}
           onSuccess={(p) => {
+            track("power-calculation-settings-update", {
+              numberOfMetrics: p.metrics.length,
```
Comment (lukesonnet) [inline packages/front-end/pages/power-calculator/index.tsx:119]: There's already a `numMetrics` used for tracking snapshot analyses. Maybe lets change the name to that (you can ctrl-f for it in another `track()` call to confirm).

File: packages/front-end/pages/power-calculator/index.tsx [hunk 2]
```diff
@@ -115,6 +115,16 @@ const PowerCalculationPage = (): React.ReactElement => {
...
+              metricsMetaData: Object.keys(p.metrics).map((m: string) => {
```
Comment (lukesonnet) [inline packages/front-end/pages/power-calculator/index.tsx:120]: Why loop over the keys and then call the object with p.metrics? I just don't know why that's the preferred approach.

In any case, looks good, we just need to ensure we can add `metricsMetaData` to our whitelist.

Comment (gazzdingo) [reply packages/front-end/pages/power-calculator/index.tsx:120]: p.metrics is an object of objects so the first is finding the object key `object.keys` the `map` is just going over the metric object to get the info we want out

**Review by lukesonnet — APPROVED** [review]

File: packages/front-end/pages/_app.tsx [hunk 3]
```diff
@@ -89,6 +89,9 @@ function App({
     growthbook.loadFeatures().catch(() => {
       console.log("Failed to fetch GrowthBook feature definitions");
     });
+    track("page-load", {
+      pathname: router.pathname,
```
Comment (gazzdingo) [inline packages/front-end/pages/_app.tsx:93]:
```suggestion
      pathName: router.pathname,
```

**Pull request merged**